src/App.tsx

```diff
@@ -1,109 +1,28 @@
 import React from 'react';
+import { PostList } from './components/PostList';
+import { Post } from './types/Post';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+
+const postsAndCommentsOfUser: Post[] = postsFromServer.map(
+  post => ({
+    ...post,
+    user: usersFromServer
+      .find(user => (post.userId === user.id)) || null,
+    comments: commentsFromServer
+      .filter(comment => (
+        comment.postId === post.id
+      )) || [],
```

Comment:
It's not easy to read this code. Would be better to create variables for user and comments